.github/actions/kots-e2e/action.yml

```diff
@@ -10,6 +10,9 @@ inputs:
   k8s-version:
     description: 'K8s version'
     required: true
+  arguments:
+    description: 'arguments to pass to k8s'
+    required: false
```

Comment:
_Really_ minor, but `arguments` is pretty generic since this isn't a k3s action. Could change this to something like `k8s-arguments` for this one? Might make it easier to know what it is when looking at the caller.